### ISSUE TITLE
Update update_data.py tqdm progress bar only on a successful stage

### ISSUE THREAD

**Issue by andrewtavis** [scribe-org/Scribe-Data#155]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/scribe-org/Scribe-Data/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow Scribe-Data's [Code of Conduct](https://github.com/scribe-org/Scribe-Data/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

As of now the [update_data.py](https://github.com/scribe-org/Scribe-Data/blob/main/src/scribe_data/wikidata/update_data.py) process allows will use a try-except to run the query [here](https://github.com/scribe-org/Scribe-Data/blob/1981f8210739429ddf515ec259a2de5083a90aae/src/scribe_data/wikidata/update_data.py#L150):

```py
try:
    results = sparql.query().convert()
except HTTPError as err:
    print(f"HTTPError with {query_path}: {err}")
```

If the except block is entered, we still update the progress bar, but actually the exception triggers a retry later in the process. Ideally the progress bar would not be updated at this stage.

### Contribution

Would be great to get some help here! Happy to support or get to it myself eventually 😊

**Comment by henrikth93** [comment]
I will take a look at it!

**Comment by andrewtavis** [comment]
Thanks, @henrikth93! As discussed, would be good to even comment out the query functionality for a moment while you work on this and then figure out how to only update the progress bar on a "successful query" :)

**Comment by axif0** [comment]
@andrewtavis, hello.. can I give a try on this?

**Comment by andrewtavis** [comment]
Hey @axif0 👋 This one is already assigned to @henrikth93. If he's fine with you taking over, then we can switch the assignment, or we can also try to find you a new issue :)

**Comment by henrikth93** [comment]
Hello @axif0 and @andrewtavis! I am looking into this. If we can not find a new issue for @axif0, then maybe we can look into it together?

**Comment by andrewtavis** [comment]
@axif0, do you want to check out #96? For this what would likely be best would be to get the articles from Wikidata for the given language and then filter them out from the result 😊 Would be happy to discuss this further!

**Comment by axif0** [comment]
Thanks @andrewtavis, I have commented there. Will you please have a look at the comment section?

**Comment by andrewtavis** [comment]
Closed by #176 :) Thanks, @henrikth93!
